CMakeLists.txt

```diff
@@ -1,17 +1,83 @@
 cmake_minimum_required(VERSION 3.10)
 
-project(cxxeffect)
+project(cxxeffect CXX)
```

Comment:
What's this do?

Reply:
I think it tells CMake that this is a C++ project as opposed to a C project